### PR TITLE
Fix zeus5 allocations

### PR DESCRIPTION
**Pull request by levity**
Zeus 5 apparently rejects mutation aliases that start with numbers

@zashton, are we doing this anywhere else? if so, please add to this branch

### CODE REVIEW

**Review by zashton — DISMISSED** [review]

**Review by topocount — COMMENTED** [review]
~~Please hold off on merging this. I've found at least one more alias that should be updated and some resultant changes that need to be made downstream~~

can't delete or dismiss my own review comment

**Review by topocount — CHANGES_REQUESTED** [review]
Please hold off on merging this. I think I've found at least one more alias that should be updated and some resultant changes that need to be made downstream.

I'll push the changes and update my review.

**Review by topocount — APPROVED** [review]
looking good now

**Pull request merged**
